### DISCUSSION TITLE
Add time asleep

### DISCUSSION THREAD

**Discussion by Carmapet** [jlai/fitness-dashboard#58] [Ideas]
Could total time asleep be added to the sleep stats?

**Comment by jlai** [comment]
Where are you looking for this? The dashboard sleep tile? Are you looking for the combined sleep time across multiple sleeps?

**Reply by Carmapet** [reply]
So sorry I totally asked for the wrong thing, must have not had enough sleep this morning.  I wanted to know if total number of awakenings during the night could be added.  It was on the Fitbit dashboard and something I looked at every morning.

**Reply by jlai** [reply]
How about this? ![sleep-awake-times](https://github.com/user-attachments/assets/2ab06475-2882-4aca-bdc8-efe8de244ac3)

**Comment by Carmapet** [comment]
That looks great.  Is there a way that it wouldn't cover the REM total time?  But yes that is what I am looking for. 

Thanks 



On Wed, 28 Aug 2024 19:42:31 -0700, Jason ***@***.***> wrote:
 
 


How about this? sleep-awake-times.png (view on web)


—
Reply to this email directly, view it on GitHub
, or unsubscribe
.

**Reply by jllyh** [reply]
Carmapet, that's just a 'hover' display. It doesn't even appear unless you hover over the 'values' region.

**Comment by Carmapet** [comment]
Thank you for replying, I didn't see it at first but now I do.  Love it and that it is on all the sleep stats!Sent from my Galaxy
-------- Original message --------From: jllyh ***@***.***> Date: 2024-08-29  1:59 p.m.  (GMT-06:00) To: jlai/fitness-dashboard ***@***.***> Cc: Carmapet ***@***.***>, Author ***@***.***> Subject: Re: [jlai/fitness-dashboard] Add time asleep (Discussion #58) 
Carmapet, that's just a 'hover' display. It doesn't even appear unless you hover over the 'values' region.
